deploy/helm/charts/templates/deployment.yaml

```diff
@@ -83,6 +83,11 @@ spec:
             # leader election is enabled.
             - name: LEADER_ELECTION_ENABLED
               value: "{{ .Values.nfsProvisioner.enableLeaderElection }}"
+            {{- if .Values.nfsProvisioner.nfsServerNamespace }}
```

Comment:
Is the values.yaml file not modified? I am not able to find it in this list of changed files.

Reply:
yes, file values.yaml is not modified.

Reply:
values.yaml is having only required params and possible supported params(not mentioned in values.yaml) are listed under deploy/helm/charts/README.md.